ios/kmbuild.sh

```diff
@@ -37,7 +37,7 @@ display_usage ( ) {
     echo "  -upload-sentry          Uploads debug symbols, etc, to Sentry"
     echo "  -debug                  Sets the configuration to debug mode instead of release."
     echo "  -download-resources     Download up-to-date versions of the engine's default resources from downloads.keyman.com."
-    echo "  -all-artifacts          Produces all build artifacts, including a Simulator-installable copy of the .app"
+    echo "  -add-sim-artifact       Produces all build artifacts, including a Simulator-installable copy of the .app"
```

Comment:
:+1: `-add-sim-artifact` is fine although artifact is a bit of a weird term. Still feel like the description could just be "Also produce a Simulator-installable .app build artifact" as otherwise it leaves me wondering "how do I build *everything except the sim artifact" but really ... it's fine.